Formula/w/wasmedge.rb

```diff
@@ -0,0 +1,35 @@
+class Wasmedge < Formula
+  desc "Lightweight, high-performance, and extensible WebAssembly runtime"
+  homepage "https://WasmEdge.org/"
+  url "https://github.com/WasmEdge/WasmEdge/releases/download/0.13.5/WasmEdge-0.13.5-src.tar.gz"
+  sha256 "95e066661c9fc00c2927e6ae79cb0d3f9c38e804834c07faf4ceb72c0c7ff09f"
+  license "Apache-2.0"
+  head "https://github.com/WasmEdge/WasmEdge.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  # llvm is required on run-time, as WASMEDGE_LINK_LLVM_STATIC defaults to OFF
+  depends_on "llvm@16"
```

Comment:
any upstream issue to support the latest llvm?

Reply:
Looks like so:
- https://github.com/WasmEdge/WasmEdge/pull/2878